keras_nlp/applications/bert.py

```diff
@@ -11,22 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Bert model and layer implementations.
-
-These components come from the tensorflow official model repository for BERT:
-https://github.com/tensorflow/models/tree/master/official/nlp/modeling
-
-This is to get us into a testable state. We should work to replace all of these
-components with components from the keras-nlp library.
-"""
+"""Bert model and layer implementations."""
 
 import tensorflow as tf
 from tensorflow import keras
 
-import keras_nlp
+import keras_nlp.layers
+
+# isort: off
+# TODO(jbischof): decide what to export or whether we are using these decorators
+# from tensorflow.python.util.tf_export import keras_export
 
+CLS_INDEX = 0
+TOKEN_EMBEDDING_LAYER_NAME = "token_embedding"
 
-class BertModel(keras.Model):
+
+# TODO(jbischof): move to keras_nlp/models
+class Bert(keras.Model):
     """Bi-directional Transformer-based encoder network.
 
     This network implements a bi-directional Transformer-based encoder as
```

Comment:
need to update bits of this docstring (all all the other exported symbols still missing docstrings)

Reply:
Other than our changes to argument names, what would you update?